expression/builtin_info.go

```diff
@@ -455,13 +455,23 @@ func (c *benchmarkFunctionClass) getFunction(ctx sessionctx.Context, args []Expr
 	// Syntax: BENCHMARK(loop_count, expression)
 	// Define with same eval type of input arg to avoid unnecessary cast function.
 	sameEvalType := args[1].GetType().EvalType()
+	// constLoopCount is used by VecEvalInt
+	// since non-constant loop count would be different between rows, and cannot be vectorized.
+	var constLoopCount int64
```

Comment:
Can we also reuse this in non-vectorized implementation?

Reply:
Sure. Codes updated, PTAL, Thanks.